src/App/App.tsx

```diff
@@ -105,6 +102,22 @@ const App: React.SFC = () => {
 
       <Switch>
 
+				<Route
+					path="/cocktails/:id"
+					render={({ match }) => {
+						const { id } = match.params;
+						return (
+							<CocktailDetails 
+								id={id} 
+								favCocktails={favCocktails} 
+								setFavCocktails={setFavCocktails}
+								toggleUserInteraction={toggleUserInteraction}
+								madeCocktails={madeCocktails}
+								setMadeCocktails={setMadeCocktails}
+							/>
+						);
+					}}
+				/>
```

Comment:
moved this route to the top of the switch since it's more specific, per Khalid's suggestion, @Melizzo @asodemann18 @LeahWinters